pkg/status/render.go

```diff
@@ -61,6 +62,9 @@ func FormatStatus(data []byte) (string, error) {
 	if traps.IsEnabled() {
 		renderStatusTemplate(b, "/snmp-traps.tmpl", snmpTrapsStats)
 	}
+	if config.IsKubernetes() {
```

Comment:
```suggestion
	if config.IsContainerized() {
```
We might want to add ECS and docker-only validation in the future.